docs/editor/profiles.md

```diff
@@ -0,0 +1,95 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+## Create a Profile
+
+You start with the Default Profile
+
+Can create an Empty Profile or one based on the current Profile.
+
+As you change settings, extension, UI state the profile is updated. Might need to refresh the Profiles view.
+
+You can delete a profile via ....
+
+### Check the current profile
+
+Title bar
+gear hover
+File > Preferences > Profiles (current profile)
+
+### Workspace Associations
+
+When you select create or select an existing profile
+
+## Profile contents
+
+Show Contents brings up the Profiles view
+
+TBD ![alt](https://) that shows the Activity bar icon
+
+### Settings
+
+### Extensions
+
+### UI state
+
+### Keybindings
+
+### Snippets
+
+### User Tasks
+
+## Share Profiles
+
+### Export
+
+As GitHub gist or local file
+
+### Import
```

Comment:
Import should go before Export.Since most users will Import, and only some will export.

Reply:
Chicken and egg.
I think we should match the ordering in the UI:

![image](https://user-images.githubusercontent.com/12818376/218603658-4df394e4-2cea-48d2-a034-a49d81f058e2.png)